packages/driver/src/cy/multi-domain/index.ts

```diff
@@ -76,11 +77,15 @@ export function addCommands (Commands, Cypress: Cypress.Cypress, cy: Cypress.cy,
         }
       }
 
-      const log = Cypress.log({
+      let log
+
+      logGroup(Cypress, {
         name: 'origin',
         type: 'parent',
         message: urlOrDomain,
-        end: true,
+        // @ts-ignore TODO: revisit once log-grouping has more implementations
```

Comment:
Is this TODO needed, or can the type be updated and if the logGroup API changes, this would then change as well?

Reply:
If the API changes, yes, it would naturally lead to changing this. But if it doesn't change, I'd like to review this and see if there's a way the API can be modified to make it less awkward for this use-case.